service-unavailable/index.html

```diff
@@ -5,13 +5,7 @@
 
     <title>Sorry, this service is unavailable - UK Export Finance</title>
     <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover" />
-    <meta name="robots" content="noindex, nofollow, noarchive, noimageindex, nosnippet" />
```

Comment:
Any reason for removing the robots?

Reply:
It's not used anywhere else. As far as I know, this `service-unavailable` is not actually used anywhere.

Reply:
I have a recollection it's used when express fails to start (I think its statically served?) I'm actually really not sure on this -- @abhi-markan can you recall?

Reply:
Resolving this - made a note to ask Abhi if we plan on doing this or if we should just remove the page.